client/boot/index.js

```diff
@@ -17,6 +17,10 @@ var React = require( 'react' ),
 	i18n = require( 'i18n-calypso' ),
 	isEmpty = require( 'lodash/isEmpty' );
 
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/degrade-es6-fns' )
+}
```

Comment:
Let's move this to the top of `Internal dependencies` or at the very top under the local-storage polyfill.